server/src/organization/organization.dto.ts

```diff
@@ -4,7 +4,9 @@ export interface OrganizationDto {
   accessCode: string;
   members: string[];
   events: string[];
+  managers: string[];
   defaultEventId: string;
+  manager_email: string;
```

Comment:
DTO looks different here, also this does not belong here. Instead, the frontend should be calling addManager to do this